### PR TITLE
Fixes media card

### PR DESCRIPTION
**Pull request by bramkragten**

### CODE REVIEW

**Review by zsarnett — APPROVED** [review]
😬

**Review by zsarnett — APPROVED** [review]

File: src/panels/lovelace/cards/hui-media-control-card.ts [hunk 1]
```diff
@@ -460,7 +464,10 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
   }
 
   private _measureCard() {
-    const card = this.shadowRoot!.querySelector("ha-card")!;
+    const card = this.shadowRoot!.querySelector("ha-card");
+    if (!card) {
+      return;
```
Comment (balloob) [inline src/panels/lovelace/cards/hui-media-control-card.ts]: This is weird. If this happens, we should probably measure again when we have a card ? So `this.updateComplete.then(() => this._measureCard())`

Comment (balloob) [reply src/panels/lovelace/cards/hui-media-control-card.ts]: Otherwise we can end up never measuring.

Comment (bramkragten) [reply src/panels/lovelace/cards/hui-media-control-card.ts]: But if it has a card, the element will be resized so `resizeObserver` will catch that case?

Comment (balloob) [reply src/panels/lovelace/cards/hui-media-control-card.ts]: 👍

**Pull request merged**